### DISCUSSION TITLE
How can you integrate Clair with ACR?

### DISCUSSION THREAD

**Discussion by DixonDalmeida** [quay/clair#1455] [Q&A]
I would like to periodically scan the ACR registry using Clair? 
How can I do it?

**Comment by hdonnay** [comment]
What have you tried?

In the broadest terms, you'd submit manifests and periodically request Index Reports.

**Reply by DixonDalmeida** [reply]
I have not tried anything yet. How can I do that?

**Reply by hdonnay** [reply]
The `clairctl` tool provides a command-line interface, you could start there.
